src/modules/sdlog2/sdlog2.c

```diff
@@ -72,6 +72,7 @@
 #include <uORB/topics/actuator_outputs.h>
 #include <uORB/topics/actuator_controls.h>
 #include <uORB/topics/vehicle_command.h>
+#include <uORB/topics/vehicle_land_detected.h>
```

Comment:
It's nice you even thought about logging, however, the landing state is already logged in the STAT message, which encodes the system state. Is that message not usable for your particular purpose? Each subscription adds overhead :).